src/connectors/ledger.js

```diff
@@ -30,42 +32,40 @@ export class LedgerConnector extends AbstractConnector {
     accountFetchingConfigs,
     baseDerivationPath
   }) {
-    super({
-      supportedChainIds: [chainId]
-    })
+    super({ supportedChainIds: [chainId] })
+
     this.chainId = chainId
     this.url = url
     this.pollingInterval = pollingInterval
     this.requestTimeoutMs = requestTimeoutMs
     this.accountFetchingConfigs = accountFetchingConfigs
     this.baseDerivationPath = baseDerivationPath
-
-    this.ledgerSubprovider = null
   }
 
-  /**
-   * @returns {Promise<Object>}
-   */
-  async activate(): Promise {
+  async activate(): Promise<ConnectorUpdate> {
```

Comment:
Uh-oh, looks like some TypeScript annotations sneaked through on this JS file.

Reply:
Weirdly enough, they didn't break my build? create-react-app / babel must have some support, _somewhere_ in those depths of hell.

Reply:
I guarantee it, yes. This is why babel makes me squeamish… I like to know what's actually going on <_<